### ISSUE TITLE
The test metric of llava_med in vqa_rad is not consistent with the metric in llava_med paper?

### ISSUE THREAD

**Issue by zh-Wang-Med** [alibaba-damo-academy/MedEvalKit#13]
Hello, I used your eval.sh script with model_name set to llava_med and model_path set to ./llava-med-v1.5-mistral-7b. However, the accuracy on closed questions in the VQA-RAD dataset was only 53, whereas the original paper reports 84 for llava-med. Could you please help identify what might be causing this discrepancy?

Also, I noticed that when running the script, it warned that the vision encoder for llava-med might not have been loaded successfully. Have you verified the correctness of the evaluation code for llava-med?

**Comment by jianghuyihei** [comment]
> Hello, I used your eval.sh script with model_name set to llava_med and model_path set to ./llava-med-v1.5-mistral-7b. However, the accuracy on closed questions in the VQA-RAD dataset was only 53, whereas the original paper reports 84 for llava-med. Could you please help identify what might be causing this discrepancy?
> 
> Also, I noticed that when running the script, it warned that the vision encoder for llava-med might not have been loaded successfully. Have you verified the correctness of the evaluation code for llava-med?

We share your confusion regarding this issue, and we're not alone. As highlighted in https://github.com/microsoft/LLaVA-Med/issues/28, many teams have reported discrepancies between their LLaVA-Med evaluation results and those published in the paper. Our hypothesis is that the results reported in the paper were likely obtained using specialized versions of the model, fine-tuned specifically on the training sets of each benchmark. For our evaluation, we used the model downloaded from https://huggingface.co/microsoft/llava-med-v1.5-mistral-7b, which appears to be the general release version.

**Comment by zh-Wang-Med** [comment]
Thanks for your reply, for question 1, we also found this issue, huatuogpt-vision reported llava-med values similar to those reported by lingshu as well, for question 2 we pulled the weights directly from hf.

**Comment by yogurt-lyn** [comment]
Sorry to bother you. Has your question 2 been resolved?

> Thanks for your reply, for question 1, we also found this issue, huatuogpt-vision reported llava-med values similar to those reported by lingshu as well, for question 2 we pulled the weights directly from hf.感谢你的回复，对于问题 1，我们也发现了这个问题，huatuogpt-vision 报告的 llava-med 值与 lingshu 报告的值相似，对于问题 2，我们直接从 hf 获取了权重。

**Comment by jianghuyihei** [comment]
> Sorry to bother you. Has your question 2 been resolved?
> 
> > Thanks for your reply, for question 1, we also found this issue, huatuogpt-vision reported llava-med values similar to those reported by lingshu as well, for question 2 we pulled the weights directly from hf.感谢你的回复，对于问题 1，我们也发现了这个问题，huatuogpt-vision 报告的 llava-med 值与 lingshu 报告的值相似，对于问题 2，我们直接从 hf 获取了权重。

You can load the checkpoint from https://huggingface.co/jianghuyihei/llava-med-v1.5-mistral-7b
